codebase/superdarn/src.lib/tk/snd.1.0/src/fitsnd.c

```diff
@@ -73,7 +73,6 @@ int FitToSnd(struct SndData *ptr, struct RadarParm *prm,
   ptr->xcf = prm->xcf;
   ptr->tfreq = prm->tfreq;
   ptr->sky_noise = fit->noise.skynoise;
-  ptr->combf = prm->combf;
```

Comment:
Should `SndSetCombf` be called in this function to perform this transfer of data correctly?

Reply:
@aburrell sorry for missing this comment!  I've kept `SndSetOriginTime` / `SndSetOriginCommand` / `SndSetCombf` all as top-level function calls from a binary in the same manner as `RadarParmSet...` in other `make_*` / `trim_*` / etc routines.